lib/cuda/cudaOutputData.cpp

```diff
@@ -30,6 +30,8 @@ cudaOutputData::cudaOutputData(Config& config, const std::string& unique_name,
         }
     }
 
+    _gpu_mem = config.get_default<std::string>(unique_name, "gpu_mem", "n2_output");
```

Comment:
this shouldn't be just hardcoded to the n2 data, should it?

Reply:
It's just a default value, but you are right, it really shouldn't have any default, that's just going to lead to bugs in config files.  Removed (in InputData as well).